src/features/swap/SwapForm.tsx

```diff
@@ -244,15 +269,52 @@ function SlippageRow() {
 
 function SubmitButton() {
   const { address, isConnected } = useAccount()
+  const { chain, chains } = useNetwork()
+  const { switchNetworkAsync } = useSwitchNetwork()
   const { openConnectModal } = useConnectModal()
+  const dispatch = useAppDispatch()
+  const { errors, touched } = useFormikContext<SwapFormValues>()
+
   const isAccountReady = address && isConnected
+  const isOnCelo = chains.some((chn) => chn.id === chain?.id)
+
+  const switchToNetwork = async () => {
+    try {
+      if (!switchNetworkAsync) throw new Error('switchNetworkAsync undefined')
+      logger.debug('Resetting and switching to Celo')
```

Comment:
should we remove this?

Reply:
It don't show by default in production so it's okay